### DISCUSSION TITLE
The DISPLAY resolution of 1536x864 causes the moonlight to fail to reach 1080p 60fps

### DISCUSSION THREAD

**Discussion by 349078838** [LizardByte/community#83] [Sunshine] [answered]
The first time I spoke on github, I used a bing translation, and please understand if there is something wrong in my speech.
I noticed a problem when I experienced it on the NVIDIA GeForce GTX 660, first looking at the dxgi-info .exe data.

D:Program FilesSunshinetools>dxgi-info.exe
====== ADAPTER =====
Device Name      : NVIDIA GeForce GTX 660
Device Vendor ID : 0x000010DE
Device Device ID : 0x000011C0
Device Video Mem : 1998 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 8147 MiB

====== OUTPUT ======
    Output Name       : .DISPLAY5
    AttachedToDesktop : yes
    Resolution        : 1536x864

====== ADAPTER =====
Device Name      : Microsoft Basic Render Driver
Device Vendor ID : 0x00001414
Device Device ID : 0x0000008C
Device Video Mem : 0 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 8147 MiB

====== OUTPUT ======
Yes, I don't know how .DISPLAY5 works, I think it is this interface that affects me to receive 1080p60fps with moonlight, up to 30fps, and I also tested the setting of moonlight to change to 720p to reach 60fps.
It feels as if the NVIDIA GeForce GTX 660's hdmi interface can only support 2k60fps or 4k30fps, not 4k60fps.
But when using NVIDIA shield, Moonlight can receive 1080p60fps, which confused me. Just like my NVIDIA GeForce GTX 660's DVI interface can support 4k60fps. (Although it is YCbCr420)
So, I'd like to ask if it is possible to modify .DISPLAY5's 1536x864 without replacing the NVIDIA GeForce GTX 660 so that Moonlight can become 1080p60fps just like with NVIDIA shield.
I myself felt it would be more appropriate to replace the NVIDIA GeForce GTX 660. And I hope I get the answer.
The following is the original text.

DISPLAY 分辨率 1536x864 导致月光无法达到 1080p 60fps
第一次在github 上发言，用的bing翻译，言语上有不对的地方还请谅解。
我在NVIDIA GeForce GTX 660上体验时发现一个问题，先看dxgi-info.exe数据。
D:\Program Files\Sunshine\tools>dxgi-info.exe
====== ADAPTER =====
Device Name      : NVIDIA GeForce GTX 660
Device Vendor ID : 0x000010DE
Device Device ID : 0x000011C0
Device Video Mem : 1998 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 8147 MiB
====== OUTPUT ======
Output Name       : \\.\DISPLAY5
AttachedToDesktop : yes
Resolution        : 1536x864

====== ADAPTER =====
Device Name      : Microsoft Basic Render Driver
Device Vendor ID : 0x00001414
Device Device ID : 0x0000008C
Device Video Mem : 0 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 8147 MiB
====== OUTPUT ======

是这样的，我并不知道\.\DISPLAY5的工作原理，我认为是这个接口影响了我用月光接收1080p60fps，最高只能接收到30fps，另外我也测试了一下月光的设置更改为720p可以达到60fps。
感觉就好像NVIDIA GeForce GTX 660的hdmi接口只能支持到2k60fps或4k30fps，不支持4k60fps一样。
但是使用NVIDIA shield的时候，月光是可以接收1080p60fps的这让我很困惑。就跟我的NVIDIA GeForce GTX 660的DVI接口可以支持4k60fps一样。（虽然是YCbCr420）
所以，我想问一下在不更换NVIDIA GeForce GTX 660的前提下，可不可以修改\\.\DISPLAY5的1536x864，使月光可以像使用NVIDIA shield一样可以变为1080p60fps。
我自己也觉得应该换掉NVIDIA GeForce GTX 660更合适一些。那也我也希望我能得到答案。

**Comment by 349078838** [comment]
Or there is another way to make Moonlight use Sunshine to reach 1080p60fps. Instead of replacing it with NVIDIA shield.
或者有别的方法，使月光可以用过Sunshine 达到1080p60fps。而不是更换为NVIDIA shield。

**Comment by ReenigneArcher** [comment, marked as answer]
That is an old graphics card. Nvidia no longer supports it with nvenc, per there compatibility matrix. https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new

With that said it might be possible to re-enable nvenc if you patch the drivers; however, I do not know if this will resolve your issue for sure.

- Linux (https://github.com/keylase/nvidia-patch)
- Windows (https://github.com/keylase/nvidia-patch/tree/master/win)

You might also try disabling the zoom on the host PC to make it 1920x1080 resolution.

**Reply by 349078838** [reply]
Understood, thank you for your answer.
明白，谢谢您的解答。

**Reply by 349078838** [reply]
I tested it, and in the GeForce GTX 660 NVIDIA shield the moonlight supports 2560*1440 60fps, compared to 1536x864, it feels outrageous.
我测试了一下，在GeForce GTX 660中NVIDIA shield中月光支持2560*1440 60fps，对比 1536x864，感觉好离谱。

**Comment by 349078838** [comment]
I tested it again with Open Stream and could pass soft decoding (only through soft decoding, all other decoding options were wrong.) It reaches 1080p60fps, but it is unstable and the latency is high.
我又使用Open Stream测试了一下，可以通过软解码（也只能通过软解码，别的解码选项都是错误。）达到1080p60fps，但是不稳定，延迟也高。

**Reply by ReenigneArcher** [reply]
Open Stream is an old, unmaintained, fork of Sunshine so I wouldn't expect anything to be drastically different.
